package.json

```diff
@@ -14,32 +14,32 @@
     "types":           "index.d.ts",
     "scripts":         {
         "build":          "webpack --config build --progress --colors --display-error-details",
-        "clean":          "rm -rf ./dist/*",
-        "dirs":           "mkdir ./dist",
-        "dist:dev":       "yarn clean && yarn go && NODE_ENV=development yarn build",
-        "dist:prod":      "yarn clean && yarn go:min && NODE_ENV=production yarn build && yarn gzip",
-        "docs":           "typedoc src/*.ts src/types/*.ts lib/types.ts",
+        "dirs":           "mkdir dist",
+        "dist:dev":       "rm -rf dist/* && yarn go && NODE_ENV=development yarn build",
+        "dist:prod":      "rm -rf dist/* && yarn go:min && NODE_ENV=production yarn build && yarn gzip",
+        "docs":           "rm -rf docs/* && typedoc src/*.ts src/types/*.ts lib/types.ts",
         "go":             "cd go/js && gopherjs build -v -o ../../lib/Amino.js .",
         "go:min":         "yarn go -m",
         "go:watch":       "yarn go -w",
-        "gzip":           "gzip -kf ./dist/*.{js,map}",
+        "gzip":           "gzip -kf dist/*.{js,map}",
         "lint":           "eslint --ext .ts,.js .",
         "nuke":           "rm -rf **/node_modules yarn.lock",
-        "prepublishOnly": "tsc && yarn lint && yarn dist:prod && yarn test",
+        "prepublishOnly": "yarn lint && yarn tsc && yarn dist:prod && yarn test",
         "reset":          "yarn nuke && yarn",
         "setup":          "yarn dirs && yarn dist:dev",
-        "test":           "jest --config ./test/jest.config.json",
+        "test":           "jest --config test/jest.config.json",
+        "tsc":            "rm -rf types/* && tsc && cp -R lib/*.d.ts types/lib/",
```

Comment:
I'd suggest using a different name for this script, e.g. `build:types`. Otherwise people might run `yarn tsc` expecting to access the TypeScript binary and accidentally run this.